src/rprblender/engine/engine.py

```diff
@@ -126,23 +126,26 @@ def depsgraph_instances(self, depsgraph: bpy.types.Depsgraph):
                 yield instance
 
     def sync_motion_blur(self, depsgraph: bpy.types.Depsgraph):
-        """ """
 
         def set_motion_blur(rpr_object, prev_matrix, cur_matrix):
-            velocity = (prev_matrix - cur_matrix).to_translation()
-            rpr_object.set_linear_motion(*velocity)
+            if hasattr(rpr_object, 'set_motion_transform'):
```

Comment:
should this just be "if pyrpr2.enabled" or something.

Reply:
Another option here to use: if isinstance(rpr_object, (pyrpr2.Shape, pyrpr2.AreaLight)).

Reply:
That seems less better.  aka worse